celery/platforms.py

```diff
@@ -229,6 +216,24 @@ def write_pid(self):
         finally:
             rfh.close()
 
+    def _is_stale(self):
+        """Check if the recorded pid in the pidfile is still alive."""
+        try:
+            pid = self.read_pid()
+        except ValueError as exc:
+            print('Broken pidfile found', file=sys.stderr)
```

Comment:
I'd prefer to use the logging module here.
What do you think?

Reply:
I think it's OK